tsl/src/compression/create.c

```diff
@@ -568,8 +568,9 @@ set_toast_tuple_target_on_compressed(Oid compressed_table_id)
 	ts_alter_table_with_event_trigger(compressed_table_id, NULL, list_make1(&cmd), true);
 }
 
+/* compression table must be created in same namespace as original table */
```

Comment:
I think this comment should be removed; as it doesn't document the method itself

I think a parameter with the name `tablespace_oid` is pretty much self explaining what it will be used for

Reply:
good point 😅 removed the comment